src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -80,5 +180,351 @@ private static Process GetProcessById(int processId)
                 return null;
             }
         }
+
+        private static string GetArgs(Process process)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                try
+                {
+                    string commandLine = WindowsProcessExtension.GetCommandLine(process);
+                    if (!String.IsNullOrWhiteSpace(commandLine))
+                    {
+                        string[] commandLineSplit = commandLine.Split(' ');
+                        if (commandLineSplit.FirstOrDefault() == process.ProcessName)
+                        {
+                            return String.Join(" ", commandLineSplit.Skip(1));
+                        }
+                        return commandLine;
+                    }
+                }
+                catch (Exception ex) when (ex is Win32Exception || ex is InvalidOperationException)
+                {
+                    return "[Elevated process - cannot determine command line arguments]";
+                }
+
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                try
+                {
+                    string commandLine = File.ReadAllText($"/proc/{process.Id}/cmdline");
+                    if(!String.IsNullOrWhiteSpace(commandLine))
+                    {
+                        //The command line may be modified and the first part of the command line may not be /path/to/exe. If that is the case, return the command line as is.Else remove the path to module as we are already displaying that.
+                        string[] commandLineSplit = commandLine.Split('\0');
```

Comment:
I don't think I understand the split on the null character here. Does /proc/pid/cmdline have null characters in it? I would have thought it would have spaces

Reply:
By convention, `/proc/<pid>/cmdline` is supposed to be a null delimited array of strings that make up the command line used to invoke the program then terminated by a null, e.g., `programname\0arg1\0arg2\0\0`. Technically, users can overwrite this memory to be anything though, e.g., Redis at one point rewrote it to a single string describing the server configuration if I recall.